apps/studio/components/interfaces/ProjectCreation/AdvancedConfiguration.tsx

```diff
@@ -58,9 +58,7 @@ export const AdvancedConfiguration = ({
                         label={
                           <>
                             Postgres
-                            <Badge color="scale" className="ml-2">
-                              Default
-                            </Badge>
+                            <Badge className="ml-2">Default</Badge>
```

Comment:
in this specific case the badges look off center
<img width="1068" height="341" alt="image" src="https://github.com/user-attachments/assets/b588f38f-7289-4a03-8fd8-03536e82444a" />